internal/apm/php_test.go

```diff
@@ -88,7 +88,7 @@ func TestPhpInjector_Inject(t *testing.T) {
 						Name: "test",
 						Env: []corev1.EnvVar{
 							{Name: "a", Value: "a"},
-							{Name: "PHP_INI_SCAN_DIR", Value: "/newrelic-instrumentation/php-agent/ini"},
+							{Name: "PHP_INI_SCAN_DIR", Value: ":/newrelic-instrumentation/php-agent/ini"},
```

Comment:
This is not correct- the default value in php.go is `/newrelic-instrumentation/php-agent/ini` without the leading `:` character. Testing with this value risks missing a mis-injected `PHP_INI_SCAN_DIR` value.

Reply:
What test case is missing?  Feel free to suggest a test case I should add.

Reply:
Apologies, I misread the test. I see that this is in the expected case, and the code changes will inject an empty string when the `PHP_INI_SCAN_DIR` is not defined in order to satisfy the new setEnvVar logic to prepend a `:`.